### PR TITLE
Update example to use cocoapods 1.0.1

### PR DESCRIPTION
**Pull request by Pearapps**
Just `pod install`s with Cocoapods 1.0.1 

### How to test

1. Open Example Project
2. Does it build and run?

### Why

Cocoapods 1.0.1 is out now so this is to make sure it works with our library.

### CODE REVIEW

File: Example/Pods/Local Podspecs/PinpointKit.podspec.json [hunk 1]
```diff
@@ -6,7 +6,7 @@
     "git": "https://github.com/Lickability/PinpointKit.git",
     "tag": "0.9"
   },
-  "summary": "A library that makes bug reporting dead simple for your testers and users by allowing them to send feedback with annotated screenshots and logs using a simple gesture.",
+  "summary": "A library that makes bug reporting simple for your users by allowing them to send feedback with annotated screenshots and logs.",
```
Comment (Pearapps) [inline Example/Pods/Local Podspecs/PinpointKit.podspec.json]: Interesting - this is not related to using the new version of Cocoapods. Just looks like the pod spec of PinpointKit was update but then no pod install happened in the example ¯_(ツ)_/¯

**Comment by mattbischoff** [comment]
We should also update the gemfile

**Comment by Pearapps** [comment]
Yup

**Comment by mattbischoff** [comment]
> Yup

And by we, I really mean you. 😎

**Comment by Pearapps** [comment]
> And by we, I really mean you. 😎

Yeah I had to do a couple errands, on it

File: Example/Pods/Local Podspecs/PinpointKit.podspec.json (hunk 1 above)
Comment (mattbischoff) [inline Example/Pods/Local Podspecs/PinpointKit.podspec.json]: Yeah, we knew about this. Part of the rush up to launch was linting this podspec and shortening this and we launched without copying it back over. nbd
